src/components/colorSwatch/index.tsx

```diff
@@ -32,6 +36,7 @@ interface Props {
   index?: number;
   style?: StyleProp<ViewStyle>;
   testID?: string;
+  size?: number;
```

Comment:
Is this a new feature? Why this is part of this PR?

Reply:
Yes, it's a new feature as the user requested.
(This feature doesn't appear in the guidelines)

Reply:
Why this is part of this PR?

Reply:
Since this feature was requested with the unavailable mode.
Not sure if I should split it into another PR.